pyanaconda/modules/storage/disk_initialization/configuration.py

```diff
@@ -32,3 +36,82 @@ def __init__(self):
         self.initialize_labels = False
         self.format_unrecognized = False
         self.clear_non_existent = False
+
+    def can_remove(self, storage, device):
```

Comment:
Why switching the name from `should_clear` to `can_remove`? AFAIK the disk which can be passed in won't be removed it will be cleared.

Reply:
I have decided to use the names `can_remove` and `can_initialize`, because we call Blivet's methods `recursive_remove` and `initialize_disk`. The cleared disks are removed and initialized from the Blivet's perspective, so these names better reflect the implementation. These methods are not going to be exposed on DBus, so it is just an implementation detail.

Reply:
Truth is it is an implementation detail so I won't block this PR because of that.
However, my point is that we shouldn't use names based on the library implementation we should go opposite way. The whole point in using the library is that we want abstraction and by naming things based on how they are implemented you are loosing this abstraction level.

Just keep this in mind, you don't need to change the code here.